homeassistant/components/vicare/binary_sensor.py

```diff
@@ -205,6 +205,7 @@ def device_info(self):
             "name": self._device_config.getModel(),
             "manufacturer": "Viessmann",
             "model": (DOMAIN, self._device_config.getModel()),
+            "configuration_url": "https://developer.viessmann.com/",
```

Comment:
Can we deep link to the device in that portal?

Reply:
Unfortunately not. The portal is very limited and only allows you to create tokens and access the docs